frontends/p4/typeChecking/typeChecker.cpp

```diff
@@ -2435,6 +2438,41 @@ const IR::Node* TypeInference::postorder(IR::Member* expression) {
         return expression;
     }
 
+    int sz = 0;
+    int size = 0;
```

Comment:
Both `sz` and size should go inside the if and I don't see why you need both.
You can do: 
```
auto size = ...
if (ht) size *= ht->width_bits()
if (member == R::Type_Header::sizeBytes) size = ROUNDUP(size, 8);
```

Reply:
Ok.  The variable in 'sz' is used by two if-statements.  First one is 'if Stack' and the other one is below it for 'if StructLike'.  Thus, I will use only 'sz' and declare it outside both if-statements.

Reply:
you do not use it outside the if body, so it can be local to each branch of the if.

Reply:
ok